website/.env.sample

```diff
@@ -27,3 +27,6 @@ NEXT_PUBLIC_WORDPRESS_API_URL=https://bubsnext.graphcdn.app
 
 # Enable bundle analyzer
 # ANALYZE=true
+
+# Vercel secret for revalidate API, also need to set this as REVALIDATE_SECRET in wp-config.php
+REVALIDATE_SECRET=bubs-next-vercel-revalidate-secret-key
```

Comment:
I think for local, this should probably be commented out? Typically we don't need to set this locally.

Reply:
updated